test/unit/ui/map.test.js

```diff
@@ -1518,30 +1728,6 @@ test('Map', (t) => {
         });
     });
 
-    t.test('no render after idle event', (t) => {
```

Comment:
These tests also look accidentally removed.

Reply:
This is just a nit now, but it looks like these two tests changed location in the file. Let's put them back where they started so it's easy to tell in the diff that nothing's changed.